physics/cu_gf_deep.F90

```diff
@@ -4742,7 +4752,7 @@ subroutine get_zu_zd_pdf_fim(kklev,p,rand_vmas,zubeg,ipr,xland,zuh2,draft,ierr,k
 
    if(zu(kpbli).gt.0.)  &
       zu(kts:min(ktf,kt-1))= zu(kts:min(ktf,kt-1))/zu(kpbli)
-     do k=my_maxloc1d(zu(:),kte),1,-1
+     do k=my_maxloc1d(zu(:),kte,1),1,-1
```

Comment:
I'm sure you'll notice this when/if you remove the `dir` variable as @dustinswales pointed out, but you can revert the addition of the extra actual arguments here and in other `my_maxloc1d` calls too.